lib/config/validation.ts

```diff
@@ -814,6 +815,23 @@ export async function validateConfig(
           ? (config.allowedHeaders as string[]) ?? []
           : GlobalConfig.get('allowedHeaders', []);
       for (const rule of val as HostRule[]) {
+        if (is.nonEmptyString(rule.matchHost)) {
+          if (rule.matchHost.includes('://')) {
+            if (is.null_(parseUrl(rule.matchHost))) {
```

Comment:
```suggestion
            if (null === parseUrl(rule.matchHost)) {
```
looks simpler than the `null_` function 🤔